sentiment_with_wordcloud.Rmd

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
should not contain a YAML header (the section at the top of the .Rmd that includes name, title, date, output, etc.) including the --- line.

Reply:
I cannot see this line "@@ -0,0 +1,89 @@" in my .rmd file . Can you please check again?

Reply:
Removed the YAML header @bl2788  Please check

Reply:
You still have that YAML header.
I did not see any changes.

Reply:
Sorry, I forgot to push the commit. Can you check now?